source/_posts/2020-05-20-release-110.markdown

```diff
@@ -0,0 +1,1068 @@
+---
+layout: post
+title: "0.110: Beta release notes"
+description: "0.110: Beta release notes"
+date: 2020-05-12 00:00:00
+date_formatted: "May 20, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-05-0.110/social.png
+---
+
+<a href='/integrations/#version/0.110'><img src='/images/blog/2020-05-0.110/integrations.png' style='border: 0;box-shadow: none;'></a>
+
+Lorem ipsum
+
+## Lorem ipsum
+
+Lorem ipsum
+
+<p class='img'>
+<img src='/images/blog/2020-05-0.110/lorem_ipsum.png' alt='Screenshot of lorem ipsum'></a>
+Screenshot of lorem ipsum.
+</p>
+
+## Icons
+
+We updated the way we load icons, with the ever growing Material Icons set, it was necessary to update the way we handle icons to make sure our application stays performant.
+We splitted the Material Icons in chunks, so you don't have to load all the icons if you just need one, besides that we no longer store the icons in the DOM but in a database.
+This saves a lot of memory and thus makes Home Assistant leaner and faster!
+
+## Weather card
+
+We got a lot of feedback about the updated weather card of the last release, we listened and added some features.
+You can now theme the colors of the new icons, and you can even completely replace them with an other image. You can also set the attribute you want to show as the secondary info.
+
+Check the [docs](/lovelace/weather-forecast/) for more info
+
+<p class='img'>
+<img src='/images/blog/2020-05-0.110/weather_card.png' alt='Screenshot of the weather card'></a>
+Screenshot of the weather card.
+</p>
+
+## Calendar panel
+
+Thanks to [@zsarnett] we got a beautiful new calendar panel! It shows you all items of the calendars you select in a month, week or day view.
+There is also a calendar card for Lovelace in the making.
+
+<p class='img'>
+<img src='/images/blog/2020-05-0.110/calendar_panel.png' alt='Screenshot of the Calendar panel'></a>
+Screenshot of the Calendar panel.
+</p>
+
+## Supervisor
+
+Have you seen the new supervisor UI? [@ludeeus] did a great job redoing it's UI! It now uses the same tabs as the configuration panel. The addon page is split into multiple tabs, the addon store is cleaned up and you can view all logs in system tab now (requires expert mode).
+
+Some advanced addons are now also marked "expert" and are only visible when you are in expert mode.
```

Comment:
Advanced mode, unless we rebranded it in 0.110 and I missed it

Reply:
Adjusted thanks!